### ISSUE TITLE
Talkers and protocols not saved to MySQL

### ISSUE THREAD

**Issue by gijs007** [ntop/ntopng#460]
I've setup ntopng community edition on Windows and it's outputting data to MySQL.
I've launched it with the command: -F "mysql;localhost;ntopng;flows;username;password"

I'm looking at the history of an interface and the chart table works fine, however the talkers and procols table result in "No Results Found"

I've also seen some screenshots of people that have a different tab for ipv4 and ipv6 flows, but I don't have that either. I'm not sure what could be wrong. I think it might be a configuration thing, but couldn't find any parameters that would enable this.

**Comment by simonemainardi** [comment]
look at the ntopng log or console output and see if there is any MySQL error. Post full ntopng configuration and make sure MySQL used specified inside -F has privileges to create and write on a databasenamed ntopng.

**Comment by gijs007** [comment]
I haven't done any configuration besides the command line for MySQL. In the preferences section on the "website" I made a few minor changes. (increasing the number of days for raw data to be saved, enabled nDPI RRD's for local hosts and networks)

I've attached the console output:
[log.txt](https://github.com/ntop/ntopng/files/189745/log.txt)

The user has all rights for the database that was created for it, data appears in the database and the chart log is working correctly.
